event_webhook_types.go

```diff
@@ -189,6 +189,7 @@ type IssueCommentEvent struct {
 		NoteableType string  `json:"noteable_type"`
 		AuthorID     int     `json:"author_id"`
 		CreatedAt    string  `json:"created_at"`
+		DiscussionID string  `json:"discussion_id"`
```

Comment:
I cannot find this field in the docs. Are you sure it's there? And additionally we try to keep the order inline with the order of the docs (or the GitLab source code), so is this the correct location for this field?

Reply:
I'm sure it's there, and it's correct order

![3gpzCA7CjB](https://user-images.githubusercontent.com/100254391/184616668-b0840c0c-e928-4122-875c-bd27e0bc9afd.jpg)